bin/adb_server.sh

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
-
-adb -P 5037 kill-server
```

Comment:
You might want to reconsider deleting this script which ensures that adb server is running. Furthermore, the relevant part of supervisor's configuration will keep trying to run this, but will fail miserably since it's deleted.